js_modules/dagster-ui/packages/ui-core/src/search/SearchResults.tsx

```diff
@@ -118,15 +221,35 @@ const List = styled.ul<ListProps>`
   padding: ${({hasResults}) => (hasResults ? '4px 0' : 'none')};
   list-style: none;
   overflow-y: auto;
+  background-color: ${Colors.backgroundDefault()};
+  box-shadow: 2px 2px 8px ${Colors.shadowDefault()};
+  border-radius: 4px;
 `;
 
 interface HighlightableTextProps {
   readonly isHighlight: boolean;
 }
 
+const MatchingFiltersHeader = styled.li`
+  background-color: ${Colors.backgroundDefault()};
+  padding: 8px 12px;
+  border-bottom: 1px solid ${Colors.backgroundGray()};
+  color: ${Colors.textLight()};
+  font-weight: 500;
+`;
+
+const SearchResultBoldedLabel = styled.span`
+  font-weight: 800;
+`;
+
+const SearchResultLabel = styled.span`
+  font-weight: 300;
+`;
```

Comment:
Generally you should not need to set `font-weight` in CSS. All of the styles used in figma should be defined here https://github.com/dagster-io/dagster/blob/f7d2611bdd46664496741b05bed3ce208d4f7b7e/js_modules/dagster-ui/packages/ui-components/src/components/Text.tsx#L35

The figma should specify whether it's Body2, Subheading2, etc.

Reply:
makes sense, updated